uint8 channel
uint32 baudrate